app.rb

```diff
@@ -0,0 +1,110 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rentals'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_all_books(_books)
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+      puts ' '
+    end
+  end
+
+  def list_all_people(_people)
+    @people.each do |person|
+      puts "[#{person.class}] name: #{person.name}, ID: #{person.id}, #{person.age}"
+    end
+  end
+
+  def create_stedent
+    print 'Age:'
+    age = gets.chomp
+    print 'name:'
+    name = gets.chomp
+    print 'Has parent permission? [Y/N]:'
+    permit = gets.chomp
+    case permit
+    when 'Y', 'y'
+      permission = true
+    when 'N', 'n'
+      permission = false
+    end
+    student = Student.new(age, name, parent_permission: permission)
+    @people.push(student)
+    puts 'Person successfully created'
+  end
```

Comment:
- [ ] Same problem still exists, When I create a new student and list all students I get unknown for the name like the image below, Please fix that

![image](https://user-images.githubusercontent.com/68796232/219882978-ec78e1b6-3882-4cc5-b7b5-6ea8a2663227.png)